<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>20</width>
      <height>600</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="nombre">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Label Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="dinero">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="dinero_label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="torre1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_torre1">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>160</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Torre1</string>
    </property>
   </widget>
   <widget class="QLabel" name="torre2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_torre2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>200</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Torre2</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>160</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Bempezar_ronda">
       <property name="text">
        <string>Empezar Ronda</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Pausa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bcerrar_sesion">
       <property name="text">
        <string>Cerrar Sesion</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Bmejora_alcance">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>260</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mejora 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bmejora_ataque">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>310</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mejora 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>390</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desarrollo de la ronda</string>
    </property>
   </widget>
   <widget class="QLabel" name="labex">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ronda:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ronda">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>370</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="enemigos_proximos">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enemigos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="puntos">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Puntos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>240</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mejora Alcance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>290</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mejora Ataque</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
